testcontainers/mssql.py

```diff
@@ -0,0 +1,42 @@
+from os import environ
+
+from testcontainers.core.generic import DbContainer
+
+
+class SqlServerContainer(DbContainer):
+    SQLSERVER_PASSWORD = environ.get("SQLSERVER_PASSWORD", "1Secure*Password1")
+    SQLSERVER_USER = "SA"
+    SQLSERVER_DBNAME = "tempdb"
+    SQLSERVER_DRIVER = "ODBC Driver 17 for SQL Server"
+
+    def __init__(self, image="mcr.microsoft.com/mssql/server:2019-latest", **kwargs):
+        super(SqlServerContainer, self).__init__(image)
+        self.port_to_expose = 1433
+        self.with_exposed_ports(self.port_to_expose)
+        self.ACCEPT_EULA = 'Y'
+        self.MSSQL_PID = 'Developer'
+
+        if 'SQLSERVER_PASSWORD' in kwargs:
```

Comment:
Can we add these as explicit parameters to the `__init__` signature? It'll make the constructor easier to understand, especially for things like [`jedi`](https://github.com/davidhalter/jedi). E.g. something like

```python
self.SQLSERVER_PASSWORD = sqlserver_password or self.SQLSERVER_PASSWORD
```

where `sqlserver_password` is an argument of the constructor.

Reply:
And done! Thank you for the lightning-fast review, I'd be glad to hear it if there's anything else!